Be clear that Attestation Result can be more than compliance/non-compliance (#137)
ietf-rats-wg/architecture: draft-ietf-rats-architecture.md

## Attestation Results

> Attestation Results are the input used by the Relying Party to decide whether it will trust a particular device, allow access to some content or data, allow a financial transaction or other.

Reviewer: ```suggestion
Attestation Results are the input used by the Relying Party to decide the extent to which it will trust a particular Attester, and allow it to access some data or perform some operation.
```

> Attestation Results may be a Boolean simply indicating compliance or non-compliance, or a rich set of Claims about the device, against which the Relying Party applies its Appraisal Policy for Attestation Results.

Reviewer: ```suggestion
Attestation Results may be a Boolean simply indicating compliance or non-compliance, or a rich set of Claims about the Attester, against which the Relying Party applies its Appraisal Policy for Attestation Results.
```
"device" -> "Attester".   Also I think this sentence is redundant with the next sentence so rather than causing duplication, I think it should be merged with the next sentence.

> Attestation Results may be a Boolean simply indicating compliance or non-compliance, or a rich set of Claims about the device, against which the Relying Party applies its Appraisal Policy for Attestation Results.

Reviewer 2: ```suggestion
Attestation Results may be a Boolean simply indicating compliance or non-compliance, a set or sequence of Boolean, or a rich set of various Claims about the device, against which the Relying Party applies its Appraisal Policy for Attestation Results.
```

> Attestation Results may be a Boolean simply indicating compliance or non-compliance, or a rich set of Claims about the Attester, against which the Relying Party applies its Appraisal Policy for Attestation Results.
> 
> Attestation Results may indicate compliance or non-compliance with a Verifier's Appraisal Policy.

Reviewer 3: ```suggestion
Attestation Results may be a Boolean simply indicating compliance or non-compliance with a Verifier's Appraisal Policy, or a rich set of Claims about the Attester, against which the Relying Party applies its Appraisal Policy for Attestation Results.

```
